news/PR-0662.rst

```diff
@@ -0,0 +1,14 @@
+**Added:**
+Exports for DAGMC CMake targets.
+
+**Changed:**
+CMake commands for linking all DAGMC libraries s.t. they are added to the exported targets.
+Included exported targets file in the generated DAGMCConfig.cmake.
```

Comment:
The news items will not be formatted properly unless there is a blank line between the title and the items. They also need bullet points ("*") at the beginning of the line or else they will all show up on the same line.